dags/data_dumps.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import airflow
+from airflow.models import DAG
+from operators.postgres_to_s3_transfer import PostgresToS3Transfer
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 3, 15),
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag = DAG(
+    dag_id='data_dumps',
+    default_args=args,
+    max_active_runs=1,
+    schedule_interval='@monthly'
+)
+
+tables_to_dump = [
+    'conditions',
+    'document_categories',
+    'documents',
+    'fda_applications',
+    'fda_approvals',
+    'files',
+    'interventions',
+    'knex_migrations',
+    'knex_migrations_id_seq',
+    'knex_migrations_lock',
+    'locations',
+    'organisations',
+    'persons',
+    'publications',
+    'records',
+    'risk_of_bias_criterias',
+    'risk_of_biases',
+    'risk_of_biases_risk_of_bias_criterias',
+    'sources',
+    'trials',
+    'trials_conditions',
+    'trials_documents',
+    'trials_interventions',
+    'trials_locations',
+    'trials_organisations',
+    'trials_persons',
+    'trials_publications',
+]
+datastore_http = airflow.hooks.BaseHook.get_connection('datastore_http')
```

Comment:
Agreed about the need to document them, not so much on the seed, as it can be tricky to do correctly. I created an issue for this: https://github.com/opentrials/opentrials/issues/696